src/olympia/amo/views.py

```diff
@@ -64,7 +64,6 @@ def client_info(request):
         'HTTP_USER_AGENT',
         'HTTP_X_COUNTRY_CODE',
         'HTTP_X_FORWARDED_FOR',
-        'HTTP_X_REQUEST_VIA_CDN',
```

Comment:
I didn't need that anymore, so might as well remove it even if the view is limited to dev.